source/faq.txt

```diff
@@ -12,6 +12,27 @@ Frequently Asked Questions
    :depth: 2
    :class: singlecol
 
+How to fix MongoServerSelectionError: connect ECONNREFUSED ::1:27017 error?
+---------------------------------------------------------------------------
+
+Beginning with version 17, Node defaults to using ``IPV6``. You have a few
+options to resolve this:
+
+- You can start your ``mongod`` with :manual:`the --ipv6 flag </reference/program/mongod/#std-option-mongod.--ipv6>`, or
+  you can set the ``net.ipv6`` option to ``true`` :manual:`in your configuration file </reference/configuration-options/#mongodb-setting-net.ipv6>`.
```

Comment:
Suggestion: since there are two solutions listed here, it could be helpful to identify which one is shown below. Alternatively, separate each bullet point into a single option.